Experimental/DistributedAuthoritySample/Assets/Scripts/Player/AvatarInteractions.cs

```diff
@@ -184,6 +197,27 @@ void OnHoldReleased(double holdDuration)
             }
         }
 
+        void CheckForPickupsInRange()
+        {
+            if(m_TransferableObject != null)
+            {
+                m_PickUpIndicator.ClearPickup();
+                return;
+            }
+
+            m_CarryBoxIndicator.HideCarry();
```

Comment:
This whole check might be refactored in the future, but this method include a visibility check, since this is invoking a coroutine every frame.

Reply:
The coroutine is not invoked it is stopped if i understand right, but still now check if the item is already hidden

Reply:
Give it a quick profile to confirm. It was being invoked every frame when just standing still.